lib/fluent/plugin/in_tail.rb

```diff
@@ -283,6 +293,22 @@ def have_read_capability?
         @capability.have_capability?(:effective, :dac_override)
     end
 
+    def extended_glob_pattern(path)
+      path.include?('*') || path.include?('?') || /\[.*\]/.match(path)
+    end
+
+    # Curly braces is not supported for now because the default delimiter of path is ",".
+    # This should be collided for wildcard pattern for curly braces.
+    def use_glob?(path)
+      if @glob_policy == :always
+        extended_glob_pattern(path) || /\{.*,.*\}/.match(path)
```

Comment:
This is because I wanted to show the possibly handled regular expressions.
If always returning true, there is no hint to debug for erroneous results from `Dir.glob(..)`.

Reply:
Hmm.
Doesn't this make it look as if there are cases where glob should not be done even when `glob_policy` is `always`?
If you want to leave information for debugging, how about using comments (instead of using codes)?

Reply:
Fair enough. So, I added comments to describe why we don't return `true` here.